docs/components/modeler/desktop-modeler/process-applications/process-application.md

```diff
@@ -0,0 +1,43 @@
+---
+id: process-application
+title: Process applications in Desktop Modeler
+description: In Desktop Modeler a process application is a folder that contains a .process-application file and a set of related files you can work on and deploy as a single bundle.
+---
+
+import OverlayImg from './img/overlay.png'
+
+In Desktop Modeler a [process application](../../../concepts/process-applications.md) is a folder that contains a `.process-application` file and a set of related files you can work on and deploy as a single bundle. These files define the automation logic but are often part of a larger project (e.g., a Java application).
+
+For instance, a consumer loan approval process application might bundle:
+
+- A main BPMN process (e.g., consumer-loan-application.bpmn) to define the workflow.
+- DMN decision tables (e.g., interest-rate-calculation.dmn, credit-score-calculation.dmn) for business rules.
+- Forms (e.g., loan-application-review.form) for user interactions.
+
+```
```

Comment:
Inside of a typical Java/Maven application the files may be structured as follows:

```suggestion
```
```